docs/framework/tools/ngen-exe-native-image-generator.md

```diff
@@ -589,9 +589,11 @@ ngen executeQueuedItems
  In the .NET Framework version 2.0, the only interaction with the native image service is through the command-line tool Ngen.exe. Use the command-line tool in installation scripts to queue actions for the native image service and to interact with the service.  
   
 ## See Also  
- [Native Image Service](http://msdn.microsoft.com/en-us/b15e0e32-59cb-4ae4-967c-6c9527781309)  
+ [Native Image Service][Native Image Service]  
```

Comment:
I'd remove this and the next link since they're inside the topic now

Reply:
done